tensorrt-sys/Cargo.toml

```diff
@@ -1,28 +1,15 @@
 [package]
 name = "tensorrt-sys"
-version = "0.3.0"
+version = "1.0.0"
```

Comment:
We don't want to bump major version to 1.0.0 here since the API is not currently stable. Pre-1.0 breaking changes require a minor version bump instead of  a major until a stable API is reached.

Appropriate version would be 0.4.0